src/crypto/CHIPCryptoPAL.cpp

```diff
@@ -1102,12 +1118,12 @@ CHIP_ERROR VerifyCertificateSigningRequestFormat(const uint8_t * csr, size_t csr
     ReturnErrorOnFailure(reader.Read8(&seq_header).StatusCode());
     VerifyOrReturnError(seq_header == kSeqTag, CHIP_ERROR_UNSUPPORTED_CERT_FORMAT);
 
-    uint8_t seq_length = 0;
+    size_t seq_length = 0;
     VerifyOrReturnError(ReadDerLength(reader, seq_length) == CHIP_NO_ERROR, CHIP_ERROR_UNSUPPORTED_CERT_FORMAT);
 
     // Ensure that outer length matches sequence length + tag overhead, otherwise
     // we have trailing garbage
-    size_t header_overhead = (seq_length <= 127) ? 2 : 3;
+    size_t header_overhead = (seq_length <= 127) ? 2 : ((seq_length <= 255) ? 3 : 4);
```

Comment:
This will give incorrect result if seq_length > 65535.

Reply:
Fixed in latest